internal/policies/fetcher.go

```diff
@@ -63,6 +63,30 @@ func (f *Fetcher) GetPoliciesForANamespace(namespace string) ([]policiesv1.Polic
 	return filteredPolicies, skippedNum, nil
 }
 
+func (f *Fetcher) getClusterAdmissionPolicies() ([]policiesv1.ClusterAdmissionPolicy, error) {
+	policies := &policiesv1.ClusterAdmissionPolicyList{}
+	err := f.client.List(context.Background(), policies)
+	if err != nil {
+		return nil, err
+	}
+	return policies.Items, nil
+}
+
+func (f *Fetcher) GetClusterAdmissionPolicies() ([]policiesv1.Policy, int, error) {
```

Comment:
please add some inline documentation explaining what the function is doing and what is returning